v3/src/components/slider/slider-component.tsx

```diff
@@ -25,6 +25,8 @@ export const SliderComponent = observer(function SliderComponent({ tile } : ITil
   const {width, height, ref: sliderRef} = useResizeDetector()
   const [running, setRunning] = useState(false)
   const animationRef = useRef(false)
+  const multiScale = layout.getAxisMultiScale("bottom")
+  const domain = layout.getAxisBounds("bottom")
```

Comment:
Is this still necessary?

Reply:
Oddly, when I don't have it in the dependency, slider value shows `NaN`